api/status.proto

```diff
@@ -28,6 +30,52 @@ message GetCephStatusResponse {
   google.protobuf.Struct progress_events = 13; 
 }
 
+message CephMonDumpResponse {
+  int32 epoch = 1;
+  string fsid = 2;
+  string modified = 3;
+  string created = 4;
```

Comment:
could it be a [timestamp](https://protobuf.dev/reference/protobuf/google.protobuf/#timestamp) (same question for `modified`)?

Reply:
`{"created": "2024-10-07T14:56:11.421751+0200",
"modified": "2024-12-06T16:32:16.198494+0100",
"last_up_change": "2024-12-03T09:19:06.401376+0100",
"last_in_change": "2024-10-07T14:56:34.891449+0200",}`
All of them are in this format, not sure if this is how we can serialize in grpc

Reply:
i think that it is worth to parce string to Go [time.Time](https://pkg.go.dev/time#Parse) and then pass it to grpc timestamp. what do you think?

Reply:
It's possible but would involve making a new struct and converting it to the grpc response seems worth it though.